js/common/util/format_date_time.js

```diff
@@ -0,0 +1,22 @@
+// Check for support of native locale string options
+function toLocaleStringSupportsOptions() {
+  try {
+    new Date().toLocaleString('i');
+  } catch (e) {
+    return e.name === 'RangeError';
+  }
+  return false;
+}
+
+export default (toLocaleStringSupportsOptions() ?
+  function (dateObj, locales) {
+    return dateObj.toLocaleString(locales, {
+      year: 'numeric',
+      month: 'long',
+      day: 'numeric',
+      hour: 'numeric',
+      minute: '2-digit',
+    });
+  } : function (dateObj, locales) { // eslint-disable-line no-unused-vars
```

Comment:
Why not just remove the unused var?

Reply:
I wanted to ensure the exported function had the same signature on both implementations.